Services/ItemServices.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Services.DTO;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Services
+{
+    public class ItemServices
+    {
+        public int GetAll(int Id)
+        {
+            return 0;
+        }
+
+        public void Save(ItemDTO itemDTO)
+        {
+           
```

Comment:
need to map ItemDTO to an Item

Reply:
Done. Kailangan pa imap yan? understanding ko sa item6 e kailangan lang imap before ipasa sa itemservices.

Reply:
Hi sir need po sa item 8. 
![image](https://user-images.githubusercontent.com/17326809/133980304-408adc11-3e61-4ff3-a65d-9621e5ad96e4.png)